### ISSUE TITLE
Add transformer for initial thread based by news

### ISSUE THREAD

**Issue by qrdwtf** [inq-surf/sm-gpt#5]

**Comment by qrdwtf** [comment]
1st version:
```json
{
    "key": "agent.news",
    "reply_to": "agent.thread",
    "input": "{\"max_tokens\": 500, \"prompt\": \"Behave as a human with following characteristics: \"{{agent}}\". You need to write a thread in a social network with your thoughts about the article with title «\"{{title}}\"»: \"{{summary}}\". Don't forget to title it.\"}",
    "output": "{\"url\": {{input.utl}}, \"title\": {{input.title}}, \"thread\": {{result.generations[0].text}}}"
}
```

**Comment by qrdwtf** [comment]
Fix #7:
```json
{
    "key": "agent.news",
    "reply_to": "agent.thread",
    "input": "{\"max_tokens\": 500, \"prompt\": \"Behave as a human with following characteristics: \"{{agent}}\". You need to write a thread in a social network with your thoughts about the article with title \\\"\"{{title}}\"\\\" and processed content: \"{{summary}}\". Don't forget to title it.\"}",
    "output": "{\"url\": {{input.url}}, \"title\": {{input.title}}, \"thread\": {{result.generations[0].text}}}"
}
```

**Comment by qrdwtf** [comment]
Update for [sm-api](https://github.com/inq-surf/sm-agent/issues/3)
```json
{
    "key": "agent.news",
    "reply_to": "agent.thread",
    "input": "{\"max_tokens\": 500, \"prompt\": \"Behave as a human with following characteristics: \"{{agent}}\". You need to write a thread in a social network with your thoughts about the article with title \\\"\"{{title}}\"\\\" and processed content: \"{{summary}}\". Don't forget to title it.\"}",
    "output": "{\"author\": \"bot\", \"source\": {{input.url}}, \"title\": {{input.title}}, \"content\": {{result.generations[0].text}}}"
}
```
